MFE/security/src/redux/sagas/certificatesSaga.js

```diff
@@ -282,6 +283,40 @@ export function* handleRegisterExternalCertificate(action) {
   }
 }
 
+export function* handleImportCertificatesInBatch(action) {
+  try {
+    yield put(loadingActions.addLoading(constants.IMPORT_CERTIFICATES_IN_BATCH));
+    const { caRoot, certificates, successCallback } = action.payload;
+
+    const caRootToBase64 = yield call(toBase64, caRoot);
+    const certificatesToBase64 = [];
+
+    Object.values(certificates).forEach(async certificate => {
```

Comment:
You are starting some promises but doesn't wait for all to end. I believe that this may cause a bug, so I recommend you to use `.map` and `Promisse.all`.